### PR TITLE
Add view quiz and overhaul the overall structure

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets:
- [#58233](https://edu-redmine.sun-asterisk.vn/issues/58233)

## WHAT this PR does?

- Add a view for user to see all the quiz and overhaul the structure to fit what I've learnt in quicktask (including renaming controller, fixing route, moving around views that should go into the same package).

## Notes Impacted Areas:
- N/A.

## Attachments:
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: resources/views/layouts/navbars/navs/auth.blade.php [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+        <a class="h4 mb-0 text-white text-uppercase d-none d-lg-inline-block" href="{{ route('quizzes.index') }}">{{ __('Quiz') }}</a>
         <!-- Form -->
         <form class="navbar-search navbar-search-dark form-inline mr-3 d-none d-md-flex ml-lg-auto">
             <div class="form-group mb-0">
                 <div class="input-group input-group-alternative">
                     <div class="input-group-prepend">
                         <span class="input-group-text"><i class="fas fa-search"></i></span>
                     </div>
-                    <input class="form-control" placeholder="Search" type="text">
+                    <input class="form-control" placeholder="Entry Code" type="text">
```
Comment (quangdm-2954) [inline resources/views/layouts/navbars/navs/auth.blade.php:13]: i18n

File: resources/views/users/index.blade.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 @section('content')
 @include('users.header', [
     'title' => __('Hello') . ' '. auth()->user()->username,
-    'description' => __('This is your profile page. You can see the progress you\'ve made with your work and manage your projects or assigned tasks'),
+    'description' => __('This is your profile page. You can see the quizzes you have taken as well as the quizzes you have created'),
```
Comment (quangdm-2954) [inline resources/views/users/index.blade.php:5]: 'description' => __('profileDescription')
en.json
profileDescription : This is your profile page. You can see the quizzes you have taken as well as the quizzes you have created

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -7,6 +7,7 @@
                 <div class="row justify-content-center">
                     <div class="col-lg-5 col-md-6">
                         <h1 class="text-white">{{ __('Welcome to Qz.') }}</h1>
+                        <h2 class="text-white">{{ __('The world smallest quiz community. Play a quiz or create your own.') }}</h2>
```
Comment (quangdm-2954) [inline resources/views/welcome.blade.php:10]: Tương tự

**Comment by ghost** [comment]
ready

**Pull request merged**